### PR TITLE
added the edge test case for `YearsBeforeDesiredBalance()`

### PR DESCRIPTION
**Pull request by hellt**
Adding the edge case when target balance has been already reached on the first iteration.
This will make student to learn checking for edge conditions

### CODE REVIEW

File: exercises/concept/interest-is-interesting/interest_is_interesting_test.go [hunk 1]
```diff
@@ -233,6 +233,12 @@ func TestYearsBeforeDesiredBalance(t *testing.T) {
 			targetBalance: 12345.6789,
 			want:          85,
 		},
+		{
+			name:          "Years before balance already reached target balance",
```
Comment (andrerfcsantos) [inline exercises/concept/interest-is-interesting/interest_is_interesting_test.go:237]:
```suggestion
			name:          "Balance is already above target",
```

**Review by andrerfcsantos — CHANGES_REQUESTED** [review]
Thanks! Suggested just a test name change.

**Review by andrerfcsantos — APPROVED** [review]
